miner/block_builder.go

```diff
@@ -29,6 +29,8 @@ const DefaultGas = 1
 
 const IncomingTxProtocol = "TxGossip"
 
+const atxsInBlockLimit = 100
```

Comment:
This is never used—remove.

Reply:
Sorry. Used.

Reply:
Ok, now I'm just confused... Why do we need a `const` in addition to a config and check that the config is smaller than the const? 🤔

Shouldn't we just trust the config? Alternatively, if we want to validate the config (convince me that we should), we should do it when loading the config, not when using the value.